### DISCUSSION TITLE
Text editor

### DISCUSSION THREAD

**Discussion by matteobisi** [community/community#3155] [Mobile] [answered]
Please include the availability of a text editor to modify at least on an iPad the files incouded in our repos

**Comment by rnystrom** [comment]
Hi! Thanks for the feedback. Would something like the simple file editor on GitHub.com be sufficient? Or are you looking for more of an IDE?

**Reply by janaka** [reply]
+1 would love to see edit functionality in the Android app.

Starting with basic edit is fine.

My use case primarily for text content rather than code. Being able to create folders and files would be awesome also. But again just edit would be a great start.

I've recently switched my blogs to Gatsby with content in MD files manages into repo on GH. Being able to make adjustments and quickly capture ideas from my mobile would be awesome.

Also, started up a Second Brain/Zettelkastan  again managed in a GH repo. I use VS Code + Foam extension for content capturing but again quickly capturing thoughts in my inbox.md and todo.md from the mobile app could be awesome.

**Comment by matteobisi** [comment]
Ho Ryan , an IDE would be an ideal condition to have all but I could see this role covered from an iOS version of vscode or similar . As start it could be great if GitHub app will implement a simple text editor to correct files on the go. I’be bought and iPad Pro and sometimes I’ve the needs to correct my repos from the iPad . Thanks 

Inviato da iPhone

**Reply by rnystrom** [reply]
Super helpful. What features do you expect from a "simple text editor"?

For example, if we simply added an "Edit" option when browsing files, would that be enough? There's a lot of file-browsing improvements we have in mind, but I'm wondering if you would need other editing features (e.g. find/replace in file, autocomplete, syntax highlighting, etc).

**Comment by matteobisi** [comment]
Hi , just edit would be awesome, but for sure if you will be able to insert edit, find and replace it would be perfect. 
 
Thanks 
                                                                                                                         Matteo Bisi                         Senior System Engineer                         
 

 
Super helpful. What features do you expect from a "simple text editor"?
For example, if we simply added an "Edit" option when browsing files, would that be enough? There's a lot of file-browsing improvements we have in mind, but I'm wondering if you would need other editing features (e.g. find/replace in file, autocomplete, syntax highlighting, etc).

**Comment by spaltrowitz** [comment, marked as answer]
This has now shipped! https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code

**Reply by matteobisi** [reply]
Thanks for the heads up! I've missed that was released, great
